### PR TITLE
Replace CSS govuk classes with nhsuk ones

### PR DESCRIPTION
**Pull request by tvararu**
This brings the interface closer in line to the prototype.

### CODE REVIEW

File: config/initializers/govuk_formbuilder.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
   #
   # https://www.rubydoc.info/gems/govuk_design_system_formbuilder/GOVUKDesignSystemFormBuilder
 
-  # config.brand: 'govuk'
-  #
+  config.brand = "nhsuk"
```
Comment (benilovj) [inline config/initializers/govuk_formbuilder.rb:6]: @tvararu does this potentially open the door to using the form builder in lots more places?

Comment (tvararu) [reply config/initializers/govuk_formbuilder.rb:6]: @benilovj I think so... I haven't tested this in anger so I don't know if it works for every single component or whether some form elements are different. We'll have to see.

Comment (tvararu) [reply config/initializers/govuk_formbuilder.rb:6]: Unfortunately, I've been looking, but the same can't be easily done for `govuk-components`. Some of the NHS components have a different HTML structure anyway, so simply changing the CSS class wouldn't make much of a difference.

**Comment by tvararu** [comment]
Unfortunately the second commit breaks the JS for the nested checkboxes, as found by the Playwright tests. I'm removing it from this PR.

image

**Pull request merged**